src/app-pages/map/components/map-details/components/time-series/TimeSeriesDateRange.js

```diff
@@ -4,27 +4,23 @@ import { connect } from "redux-bundler-react";
 import { radarTimeControls } from "../../../../../../app-bundles/radar-time-series-bundle";
 import { dateToString } from "../../../../../../utils";
 
-const TimeSeriesDateRange = ({
+export const TimeSeriesDateRange = ({
   ltsTimeControl,
   ltsCustomStartDate,
   ltsCustomEndDate,
   doLtsSetTimeControl,
   doLtsSetCustomDate,
 }) => {
   const handleDateRangeOnChange = (e) => {
-    e.stopPropagation();
     doLtsSetTimeControl(parseInt(e.target.value));
   }
 
   const handleDateOnChange = (e) => {
-    e.stopPropagation();
     const name = e.target.name;
     const date = e.target.value;
     doLtsSetCustomDate(name, date);
   };
 
-  const handleOnClick = e => e.stopPropagation();
```

Comment:
Wasn't this here to stop the sidebar from closing when clicking on the fields? Or did you stop that from happening some other way?

Reply:
I removed that functionality, I thought it was unintuitive to close the sidebar if you click on the sidebar. Now the sidebar will only close if you click the collapse arrow button.